cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help

```diff
@@ -1,7 +1,5 @@
 NAME
-       ballerina-graphql - Generate the Ballerina client sources for a GraphQL config file, 
-       generate the GraphQL schema for a Ballerina GraphQL service,
-       and generate the Ballerina service sources for a GraphQL schema.
+       ballerina-graphql - Generate the Ballerina client sources for a GraphQL config file, the GraphQL schema for a GraphQL service, and vice versa.
```

Comment:
"and vice versa" is ambiguous here. Better to clearly state "generate GraphQL service from a schema".

Reply:
Won't it be too long for the short description as we include a longer description also?

Reply:
But shouldn't we prioritise the clarity here? We have three functionalities in the GraphQL tool. We are mentioning two, and fill it with vice-versa which I find ambiguous.

Reply:
Updated.